devices/wholeBodyDynamics/README.md

```diff
@@ -46,6 +46,9 @@ For an overview on `wholeBodyDynamics` and to understand how to run the device,
 |                      | enableGravityCompensation | bool | -  | -           | No        |  |  |
 |                      | gravityCompensationBaseLink| string | - | -         | No        | ..  | |
 |                      | gravityCompensationAxesNames | vector of strings | - | - | No   | Axes for which the gravity compensation is published. | |
+| HW_USE_MAS_IMU |  -  | group             | -     | -     | No       |  Group to enable attach to Multiple Analog Sensor interface based IMU. | If the group is not present, the default behavior of attaching to IGenericSensor interface IMu will be carried out. |
```

Comment:
```suggestion
| HW_USE_MAS_IMU |  -  | group             | -     | -     | No       |  Group to enable attach to Multiple Analog Sensor interface based IMU. | If the group is not present, the default behavior of attaching to IGenericSensor interface IMU will be carried out. |
```

Reply:
Fixed in https://github.com/robotology/whole-body-estimators/pull/99/commits/5dd8c48392d5abf0e48500c4870540663db0cbec